docs/docs/.pages

```diff
@@ -0,0 +1,6 @@
+nav:
```

Comment:
This is a way how to reorder how the pages are displayed in the navigation.

It is using awesome pages plugin - https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin#customize-navigation